### ISSUE TITLE
Latest version on Flathub is v3.22

### ISSUE THREAD

**Issue by deusnovus** [lassekongo83/adw-gtk3#238]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/lassekongo83/adw-gtk3/issues?q=is%3Aissue) to avoid opening a duplicate issue.

### Checklist to make sure the issue is not on your end

- [X] I have made sure that I do not have multiple versions of the theme installed. Only one version should be installed in either `~/.local/share/themes` or `/usr/share/themes` or `~/.themes`
- [X] The file `~/.config/gtk-3.0/gtk.css` does not exist. (If it does, backup and remove it.)
- [X] I have logged out and in again.
- [ ] Dark theme only. I have enabled adw-gtk3-dark in `gnome-tweaks` AND set the appearance to dark in `gnome-control-center`
- [X] The flatpak theme is installed `org.gtk.Gtk3theme.adw-gtk3` and `org.gtk.Gtk3theme.adw-gtk3-dark`
- [X] The app I'm having an issue with is an actual GTK3 app and is not coded using another toolkit like GTK2, GTK4, QT, etc.
- [X] I have created a new user profile without any modifications except adw-gtk3.

### Description

I installed adw-gtk3 via the flatpak command line in the repo's README.md and for some reason, its latest version seems to be v3.22, as seen below. I performed `sudo dnf upgrade` and `sudo dnf update` for good measure, but that's about it. As expected, most gtk3 apps installed via Flathub have little to no modifications.

![adwgtk3png](https://github.com/lassekongo83/adw-gtk3/assets/32219080/12b96d2b-ceb2-4037-beff-cb0fd6d3aae7)

### OS and version [e.g Fedora 37]

Fedora 38

### Adw-gtk3 version. Also specify your installation source. [e.g tar.xz, AUR, git main branch]

3.22

### GNOME version

44

**Comment by deusnovus** [comment]
Yes, I'm also very confused. My repos and all other flatpak apps seem to be up to date. Just to confirm: is the latest flatpak version supposed to be v.4.8?

**Comment by ozky84** [comment]
Flatpak release number is not updated it still show's 3.22 even it's build from 4.8 sources..

[Flatpak](https://github.com/flathub/org.gtk.Gtk3theme.adw-gtk3/blob/master/org.gtk.Gtk3theme.adw-gtk3.yaml)

**Comment by deusnovus** [comment]
I see, that's interesting... This probably has no bearing on the theme's performance itself. I still have some GTK3 apps like Geary and Inkscape that are still not modified by adw-gtk3, so I was worried I was on an older version, but perhaps that's not the case.
